### DISCUSSION TITLE
metascore imdb

### DISCUSSION THREAD

**Discussion by eeinnew** [teamalgoritma/community#50] [Q&A] [answered]
Selamat Pagi Pak/Bu, saya ingin bertanya terkait capstone saya,
saya memiliki kendala di metascore,
dimana ada 2 jenis data metascore yaitu yang warna hijau(metascore favorable) dan kuning (metascore mixed)

yang aku bingung bagaimana bisa mengambil data yang metascore mixed, saya sudah coba pake fungsi or dan and di classnya tapi ttp tidak muncul
<img width="960" alt="image" src="https://user-images.githubusercontent.com/81936292/163754991-1b20e1b1-3537-44b5-9346-94902e320889.png">

**Comment by triarm** [comment, marked as answer]
Hallo Bu Wennie,
terkait hal tersebut ibu bisa membuat 3 buah kondisional: `metascore favorable == None`, `metascore mixed == None`, serta ketika keduanya `None`.
```
favorable = cont[i].find('span', attrs={"class": "metascore favorable"})
mixed = cont[i].find('span', attrs={"class": "metascore mixed"})
# kondisi ketika keduanya None
if (favorable == None) and (mixed == None):
    meta = 0
# kondisi ketika favorable None
elif favorable == None:
    meta = mixed.text
# kondisi ketika mixed None
elif mixed == None:
    meta = favorable.text
```

**Reply by eeinnew** [reply]
halo pak, ini kenapa jadi hanya kondisi 1 yang terjalan ya, soalnya semuanya metascorenya jadi 0
<img width="960" alt="image" src="https://user-images.githubusercontent.com/81936292/163807901-140c02bc-bbfd-4305-9405-da5740da77e8.png">

**Reply by triarm** [reply]
Untuk elemen ketiga pada setiap tuplenya (74,71,74,...) bukankah itu sudah metascorenya bu?

**Comment by eeinnew** [comment]
oh iya pak makasih sudah bisa
